[2022-12-06 14:28:34,943] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:28:18.765624+00:00 [queued]>
[2022-12-06 14:28:34,966] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:28:18.765624+00:00 [queued]>
[2022-12-06 14:28:34,966] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 14:28:34,966] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-12-06 14:28:34,967] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 14:28:34,977] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): LoadPositionsInMongoDB> on 2022-12-06T14:28:18.765624+00:00
[2022-12-06 14:28:34,982] {standard_task_runner.py:52} INFO - Started process 205 to run task
[2022-12-06 14:28:34,987] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_mongo_db_dag', 'LoadPositionsInMongoDB', '2022-12-06T14:28:18.765624+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_mongo_db_dag.py', '--cfg-path', '/tmp/tmpr3wofv9f', '--error-file', '/tmp/tmpfa7y66bf']
[2022-12-06 14:28:34,987] {standard_task_runner.py:77} INFO - Job 9: Subtask LoadPositionsInMongoDB
[2022-12-06 14:28:35,051] {logging_mixin.py:104} INFO - Running <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:28:18.765624+00:00 [running]> on host db4794db4ef9
[2022-12-06 14:28:35,137] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_mongo_db_dag
AIRFLOW_CTX_TASK_ID=LoadPositionsInMongoDB
AIRFLOW_CTX_EXECUTION_DATE=2022-12-06T14:28:18.765624+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-06T14:28:18.765624+00:00
[2022-12-06 14:28:35,359] {logging_mixin.py:104} INFO - debut de chargement des données dans mongodb
[2022-12-06 14:28:35,710] {logging_mixin.py:104} INFO - fin de chargement des données dans mongodb
[2022-12-06 14:28:35,716] {python.py:151} INFO - Done. Returned value was: None
[2022-12-06 14:28:35,732] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_mongo_db_dag, task_id=LoadPositionsInMongoDB, execution_date=20221206T142818, start_date=20221206T142834, end_date=20221206T142835
[2022-12-06 14:28:35,768] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-06 14:28:35,801] {local_task_job.py:151} INFO - Task exited with return code 0
